src/vs/platform/terminal/common/capabilities/capabilities.ts

```diff
@@ -155,6 +155,9 @@ export interface IShellEnvDetectionCapability {
 	readonly onDidChangeEnv: Event<Map<string, string>>;
 	get env(): Map<string, string>;
 	setEnvironment(envs: { [key: string]: string | undefined } | undefined, isTrusted: boolean): void;
+	startSingleEnvironmentVar(isTrusted: boolean): void;
+	setSingleEnvironmentVar(key: string, value: string | undefined, isTrusted: boolean): void;
+	endSingleEnvironmentVar(isTrusted: boolean): void;
```

Comment:
Let's rename to `*EnvironmentSingleVar` to be a little closer to `setEnvironment`

Reply:
Resolved in https://github.com/microsoft/vscode/pull/237602/commits/9032f1646b0b3f8ea9b092e5ad087b21590e0c70